editor/src/components/assets.ts

```diff
@@ -12,6 +12,7 @@ import Utils from '../utils/utils'
 import * as R from 'ramda'
 import { dropLeadingSlash } from './filebrowser/filepath-utils'
 import { fastForEach } from '../core/shared/utils'
+import { mapValues } from 'xstate/lib/utils'
```

Comment:
Did you mean to import this function or the one defined in `../core/shared/object-utils`?

Reply:
What on earth?

Reply:
Fixed in 477c7b28.